src/pages/NIHicWebform.js

```diff
@@ -665,1250 +289,1578 @@ class NIHICWebform extends Component {
-                        disabled: false,
-                      }),
-
-                      RadioButton({
-                        style: {
-                          marginBottom: '2rem',
-                          marginLeft: '2rem',
-                          color: ' #010101',
-                        },
-                        id: 'checkPublicAccess_no',
-                        name: 'checkPublicAccess',
-                        label: 'No',
-                        disabled: false,
-                      }),
-                    ]),
-                  ]),
+                  div(
+                    {className: 'col-xs-12 rp-group rp-last-group'}, [
+                      input({
+                        type: 'text',
+                        name: 'assistantEmail',
+                        id: 'assistantEmail',
+                        placeholder: 'email@domain.org',
+                        maxLength: '256',
+                        className: 'form-control',
+                        required: true,
+                      })
+                    ])
                 ]),
 
                 div({className: 'form-group'}, [
                   div(
-                    {className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'},
-                    [
+                    {className: 'col-xs-12 rp-group'}, [
                       label({className: 'control-label rp-title-question common-color'}, [
-                        'The genomic summary results (GSR) from this study are only to be made available through controlled-access',
+                        'Authenticate with eRA Commons',
                       ]),
                     ]),
-                  div({ className: 'row no-margin' }, [
-                    div({ className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 dataset-group' }, [
-
-                      RadioButton({
-                        style: {
-                          margin: '2rem',
-                          color: ' #010101',
-                        },
-                        id: 'checkPublicAccess_yes',
-                        name: 'checkPublicAccess',
-                        label: 'Yes',
-                        disabled: false,
-                      }),
-
-                      RadioButton({
-                        style: {
-                          marginBottom: '2rem',
-                          marginLeft: '2rem',
-                          color: ' #010101',
-                        },
-                        id: 'checkPublicAccess_no',
-                        name: 'checkPublicAccess',
-                        label: 'No',
-                        disabled: false,
-                      }),
-                    ]),
-                  ]),
+                  eRACommons({
+                    className: 'col-sm-6 col-xs-12 rp-group',
+                    destination: null,
+                    onNihStatusUpdate: () => {},
+                    location: location
+                  })
```

Comment:
This component requires an auth-ed user to be logged in so any non-auth-ed user will be redirected back to the home page. If this page needs to be public, then we need to replace this with a free text field for the user to enter their era commons id.